frontend/apps/reader/modules/readermenu.lua

```diff
@@ -27,21 +27,27 @@ function ReaderMenu:init()
         -- items in top menu
         navi = {
             icon = "appbar.navigation",
+            text = _("Navigation"),
         },
         typeset = {
             icon = "appbar.typeset",
+            text = _("Typeset"),
```

Comment:
```suggestion
            text = _("Typesetting"),
```

Reply:
But actually if you want to name it for end users:

```suggestion
            text = _("Format"),
```

```suggestion
            text = _("Formatting"),
```